core/src/main/java/org/elasticsearch/index/query/GeoDistanceQueryBuilder.java

```diff
@@ -19,115 +19,299 @@
 
 package org.elasticsearch.index.query;
 
+import com.google.common.base.Preconditions;
```

Comment:
As we are currently trying to get rid of guava as a dependency - switch from using Preconditions to an explicit check?